### PR TITLE
Updated to new FXv2 Style fxmanifest.lua files

### PR DESCRIPTION
**Pull request by Rosw3lluk**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -29,7 +29,15 @@ ResourceManifestPlugin.prototype.apply = function(compiler) {
 
 function format(assets, path) {
   return `
-ui_page "${path}/index.html"
+  name 'resource name'
```
Comment (calumari) [inline index.js]: Can you remove the indentation please? It causes the generated manifest to have indentations too.

File: index.js [hunk 2]
```diff
@@ -29,7 +29,15 @@ ResourceManifestPlugin.prototype.apply = function(compiler) {
...
+  fx_version 'cerulean'
+  games { 'gta5', 'rdr3 }
+  
+  shared_scripts { 'config.lua' }
```
Comment (calumari) [inline index.js]: Any chance you could remove shared, client and server scripts please? IIRC, I tried to keep the generated manifest fairly basic and not tied any project's structure.

**Review by calumari — CHANGES_REQUESTED** [review]
Hi, thanks for the PR! I haven't been involved in FiveM in a while and have neglected this project. I believe https://github.com/BlueSky-Development/webpack-fx-manifest is an updated fork. I ran your changes and left a couple of comments.

**Comment by Rosw3lluk** [comment]
Hopefully the changes have applied and are to your satisfaction :)

**Comment by calumari** [comment]
LGTM, thanks!

**Pull request merged**